.github/workflows/notify-on-push.yml

```diff
@@ -0,0 +1,28 @@
+on:
+  push:
+    branches:
+      - main
+
+name: Notify on Push
+permissions:
+  contents: read
+
+jobs:
+  notify_on_push:
+    name: Notify on Force Push on `main`
+    if: github.repository == 'nodejs/nodejs.org' && github.event.forced
+    runs-on: ubuntu-latest
+    steps:
+      - name: Slack Notification
+        uses: rtCamp/action-slack-notify@c33737706dea87cd7784c687dadc9adf1be59990 # 2.3.2
+        env:
+          SLACK_COLOR: '#DE512A'
```

Comment:
```suggestion
          SLACK_COLOR: '#DE512A' # orange 
```

Reply:
I purposely kept it the same as the nodejs/node repo - but 🤷

Reply:
yeah  I just propose to explain which colors is behind this exa